scripts/simplify_network.py

```diff
@@ -285,10 +292,16 @@ def simplify_links(n, costs, config, output, aggregation_strategies=dict()):
             pass
         return n, n.buses.index.to_series()
 
+    dc_as_links = not n.links.loc[n.links.carrier == "DC"].empty
+    dc_as_lines = not n.lines.loc[n.lines.carrier == "DC"].empty
```

Comment:
were you looking for the flag in config maybe? In case, you could pass it as a parameter here, what do you think?

Reply:
Yeah, I have been thinking about taking a flag directly from the config, but decided to try reading it from the network itself. The idea was to make it safer, although agree that it might be not clear. 

Happy to revise if you think it'd be better to used a flag from the config

Reply:
Mah... agree that we could avoid overcomplicating the code.
We could do the check internally, however, it may be safer to check for dc lines: 
`dc_as_links = not (n.lines.carrier == "DC").any()`
and implicitly assume that when it is false than we used lines

As even when lines are included, converters represented as links will be necessary as well, therefore, with the current formulation dc_as_links will always be true.

the if below where you check: `if dc_as_links: .... elif dc_as_lines`, may be revised with: `if dc_as_links: ... else: ...` so replacing the elif with else.

I'm unsure if some issues may be fixed with this but i'm unsure

Reply:
That is definitely cleaner :)
Thanks for explanations! Done